Dockerfile

```diff
@@ -1,14 +1,29 @@
-FROM ghcr.io/tiiuae/fog-ros-baseimage-builder:v2.1.0
+FROM ros:humble-ros-base as builder
```

Comment:
What is the reason to not use the builder/SDK image?
We should use the SDK image so we will have aligned all the compiler and library versions used on runtime.

Reply:
This builds will not be used for the containers that utilize the fog-ros-baseimage, but for the development environments. For instance, one can install this deb package (`ros-humble-px4-msgs`) in the local linux machine on amd64, ubuntu based Jetson (arm64), or a container with the ros:humble-ros-base build. Therefore, I replaced it to use the ros:humble-ros-base. The actual build in our system is still done with the yocto.

Reply:
How can we document this?